### PR TITLE
⚰️ zb: remove unused arg0namespace from match rule

### PR DESCRIPTION
**Pull request by MaxVerevkin**
It was replaced by arg0ns and is now unused.

### CODE REVIEW

File: zbus/src/match_rule/builder.rs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-use std::convert::{TryFrom, TryInto};
+use std::convert::TryInto;
```
Comment (zeenix) [inline zbus/src/match_rule/builder.rs:1]: Now that we've `2021` edition enabled, you can just drop this whole line I think.

**Review by zeenix — APPROVED** [review]
Looks good. @elmarco broke the Windows CI so either he'll fix that or I'll revert his commits soon.

File: zbus/src/match_rule/builder.rs (hunk 1 above)
Comment (MaxVerevkin) [reply zbus/src/match_rule/builder.rs:1]: Awesome

**Review by zeenix — APPROVED** [review]
Thanks

**Pull request merged**